email/mjml_service/handler.py

```diff
@@ -6,22 +6,19 @@
 import socket
 import requests
 
-class MJMLHandler:
-    def __init__(self, ip):
-        self.ip = ip
-
-    def convert(self, template_name='beforeLiveCodeReview'):
-        data = {
-            'templateName':template_name,
-            'templateVariables': {'title':'TYTUŁ POSTA'}
-        }
-        MJML_IP = f'http://{self.ip}:3000/convert'
-        try:
-            resp = requests.post(MJML_IP,  json=data)
-            return resp.json()
-
-        except requests.exceptions.RequestException as e:  
-            return {'error':str(e)}
+
+def MJMLHandler_converter(ip, template_name='beforeLiveCodeReview'):
```

Comment:
po co robić funkcję jak jest gotowy handler ?
zrób sobie statyczną funkję convert i wyjdzie na to samo

Reply:
zamieniłem obiekt na funkcje, obiekt w tym przypadku to overkill

Reply:
czy ja wiem, handler na sender do maila będzie musiałbyć klasą (bo będzie mieć wiele akcji), więc dlatego myślę że handlery powinny być spójne w swojej strukturze

Reply:
ok, i tak musiałem to wykomentować więc można, póki co nie wpłynęło to na działanie w żaden sposób